### ISSUE TITLE
Circular import in pyramid.viewderivers

### ISSUE THREAD

**Issue by philpep** [Pylons/pyramid#3112]
Hello, importing `pyramid.viewderivers` trigger a circular import error:

```
>>> import pyramid.viewderivers
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File ".../lib/python2.7/site-packages/pyramid/viewderivers.py", line 31, in <module>
    from pyramid.config.util import (
  File ".../lib/python2.7/site-packages/pyramid/config/__init__.py", line 78, in <module>
    from pyramid.config.views import ViewsConfiguratorMixin
  File ".../lib/python2.7/site-packages/pyramid/config/views.py", line 80, in <module>
    from pyramid.viewderivers import (
ImportError: cannot import name INGRESS
```

**Comment by mmerickel** [comment]
I guess that's what happens when you have 2 util files!

**Comment by mmerickel** [comment]
Fixed in 1.8.5 and 1.9.1.

**Comment by philpep** [comment]
Thanks !
